lib/Sema/SemaBounds.cpp

```diff
@@ -1115,7 +1115,7 @@ namespace {
 
   public:
     CheckBoundsDeclarations(Sema &S) : S(S),
-     DumpBounds(S.getLangOpts().DumpInferredBounds) {}
+      DumpBounds(S.getLangOpts().DumpInferredBounds) {}
```

Comment:
This change is fixing the spacing - it should have been two spaces to start with.